test/gui/shared/steps/steps.py

```diff
@@ -448,7 +450,7 @@ def step(context, resource):
     openSharingDialog(context, resource)
     publicLinkDialog = PublicLinkDialog()
     publicLinkDialog.openPublicLinkDialog()
-    publicLinkDialog.createPublicLink(context, resource, '', '', '')
+    publicLinkDialog.createPublicLink(context, resource, '', '')
```

Comment:
there must be a way to pass default args in python so you don't have to give empty strings here.